app/reducers/dimensions.js

```diff
@@ -0,0 +1,23 @@
+import { DIMENSIONS } from '../actions/actionsTypes';
+
+const initialState = {
```

Comment:
We're about to remove `window` and `screen` objects, so you can create these values as the keys returned from `Dimensions.get('window')`.